app/models/poi.rb

```diff
@@ -2,8 +2,15 @@ class Poi < ApplicationRecord
   validates :name, presence: true
   validates :x, presence: true, numericality: { only_integer: true , greater_than: 0}
   validates :y, presence: true, numericality: { only_integer: true , greater_than: 0}
-
   def coords
     {x: x, y: y}
   end
+
+  def method_name
+    if true
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.